tests/testthat/test-5runNMDS.R

```diff
@@ -8,7 +8,7 @@ test_that("addNMDS", {
     actual <- getNMDS(se)
     expect_true(is.matrix(actual))
     expect_equal(dim(actual),c(10,2))
-    actual2 <- getNMDS(se,nmds="monoMDS",pc=FALSE,scaling=FALSE)
```

Comment:
Yes. since nmdsFUN had the prefix, the test always passed. but now we have multiple parameter with nmds prefix which caused the test error because R didn't know which param to match to.

Reply:
Thanks for clarifying